src/installer/corehost/cli/apphost/static/CMakeLists.txt

```diff
@@ -46,11 +55,50 @@ include(configure.cmake)
 add_definitions(-DFEATURE_APPHOST=1)
 add_definitions(-DFEATURE_STATIC_HOST=1)
 
-# Disable manifest generation into the file .exe on Windows
 if(CLR_CMAKE_TARGET_WIN32)
-    set_property(TARGET ${PROJECT_NAME} PROPERTY
-            LINK_FLAGS "/MANIFEST:NO"
+    # Disable manifest generation into the file .exe on Windows
+    add_linker_flag("/MANIFEST:NO")
```

Comment:
I do not think we need a general solution here. This will not be needed after we move hosts to clr partition (tracked by #43700).

We just need to finish superhost - i.e. enable it not only on Linux. 
Then we should move the entire thing together with tests and required infra to clr partition where it will not be possible to have different configs for the native parts.

Reply:
As for mixed configurations - I think at least combinations with Checked runtime are useful, for the same reason they are useful with libs - running installer tests with Debug runtime is very slow.